PharoLauncherTutorial/pharolauncher_tutorial.pillar

```diff
@@ -0,0 +1,130 @@
+
+! PharoLauncher Tutorial: What ? How ?
+
+
+This unit aims to provide a basic overview of what PharoLauncher is and give a step-by-step approach on how to use this application of great advantage. 
+
+
+!!Overview
+
+Pharo is an open source implementation of the programming language and environment Smalltalk. Pharo is not Smalltalk. Pharo is Smalltalk-inspired. 
+
+Pharo offers strong live programming features such as immediate object manipulation, live update, and hot recompilation. Live programming environment is in the heart of the system. Pharo also supports advanced web development with frameworks such as Seaside and more recently Tide. 
+
+The official Pharo website defines it as: ''Pharo is a pure object-oriented programming language and a powerful environment, focused on simplicity and immediate feedback (think IDE and OS rolled into one). ''
+
+Pharo relies on a virtual machine that is written almost entirely in Smalltalk itself. 
+
+Pharo environment is its own little world, designed around a conception of a computer with a minimal operating system and populated with living objects. A Smalltalk implementation is composed of an image (binary code), a major source file and a ‘changes’ file. The image is called Virtual Image (VI) because is independent form the platform you use for running Smalltalk. Smalltalk systems store the entire program state (including both Class and non-Class objects) in an image file. The image can then be loaded by the Smalltalk virtual machine to restore a Smalltalk-like system to a prior state. 
+
+As Pharo is open source, it growing rapidly owing to the contributions of people all around the world. Each day we have a new update of the image of Pharo which makes it cumbersome to keep track of updates. It becomes quite a task when one has to download a new image seperately each he/she plans to work on something having the latest issues fixed, new features added etc.
+
+That's where the PharoLauncher comes in the picture.
+
+Pharo Launcher, a cross-platform application that 
+- lets you manage your Pharo images (launch, rename, copy and delete); 
+- lets you download image templates (i.e., zip archives) from many  different sources (Jenkins, files.pharo.org, and your local cache); 
+- lets you create new images from any template.
+
+The idea behind the Pharo Launcher is that you should be able to access it very rapidly from your OS application launcher. As a result launching any image is never more than 3 clicks away. 
+
+"PharoLauncher" is useful to a user who develops and needs to constantly switch between images. PharoLauncher is also a very handy tool to download specific image update versions if you want to reproduce or fix Pharo bugs.Pharo Launcher is a Pharo-based application allowing you to manage a list of images (download, rename, delete) and switch between them without aditional tools. 
+
+
+
+!! Downloading/Installing PharoLauncher
+
+!!!Linux Ubuntu:
```

Comment:
Since things are changing rapidly here (we're updating the Linux instructions currently), maybe just put a link to http://www.pharo.org/download explaining that by following the instructions here will install Pharo Launcher